src/coreclr/inc/stresslog.h

```diff
@@ -772,7 +797,7 @@ class ThreadStressLog {
 
     BOOL IsValid () const
     {
-        return chunkListHead != NULL && (!curWriteChunk || curWriteChunk->IsValid ());
+        return chunkListHead != NULL && (!curWriteChunk || StressLog::TranslateMemoryMappedPointer(curWriteChunk)->IsValid ());
```

Comment:
This needs a bit of explanation. The address translation is needed only for the stress log analyzer and this seems to be the only one that the stress log analyzer calls into and performs a pointer dereferencing.